### DISCUSSION TITLE
with what source to merge this independant git ?

### DISCUSSION THREAD

**Discussion by firelook** [celguar/mangosbot-bots#70] [Q&A]
please 
with what directory source and version number this independant "addons" is "plug and play" ?

thanks !

**Comment by celguar** [comment]
@firelook Hello! I'm currently updating all 3 core repos with some performance tweaks to allow more smooth experience with lots of random bots. I hope to finish it tonight, after that you will be able to build them. I'm sorry for inconveniences 😏

**Comment by firelook** [comment]
Wonderfull news !!!!
And please at which version you plan to mach Cmangos official repo => it seems you are nearly 3 months old ?
thanks again you do a great job !

**Reply by celguar** [reply]
Hello, I plan to sync with latest cmangos within a week or two, I also need to update repack's launcher a bit to apply all new changes for users 😊

**Reply by celguar** [reply]
@firelook Hello, core repos are updated and bots submodules are in sync now. Should be buildable :)
Update for latest cmangos will come a bit later
